CDCgov/prime-reportstream
# Feature/sync terraformstate

This PR is to sync the terraform state with code in staging environment

Test Steps:
1. Run terraform plan in each module and verify any outstanding changes from code 

## Changes
- Function app module
-Database module
- log analytics workspace
- storage account
